model_compression_toolkit/core/common/mixed_precision/bit_width_setter.py

```diff
@@ -36,35 +36,40 @@ def set_bit_widths(mixed_precision_enable: bool,
 
     """
     if mixed_precision_enable:
-        assert all([len(n.candidates_quantization_cfg) > 0 for n in graph.get_configurable_sorted_nodes()]), \
+        assert all([len(n.candidates_quantization_cfg) > 0
+                    for n in graph.get_configurable_sorted_nodes(graph.fw_info)]), \
             "All configurable nodes in graph should have at least one candidate configuration in mixed precision mode"
 
         Logger.info(f'Set bit widths from configuration: {bit_widths_config}')
         # Get a list of nodes' names we need to finalize (that they have at least one weight qc candidate).
-        sorted_nodes_names = graph.get_configurable_sorted_nodes_names()
-        for node in graph.nodes:  # set a specific node qc for each node final weights qc
+        sorted_nodes_names = graph.get_configurable_sorted_nodes_names(graph.fw_info)
+        for node in graph.nodes:  # set a specific node qc for each node final qc
             # If it's reused, take the configuration that the base node has
             node_name = node.name if not node.reuse else '_'.join(node.name.split('_')[:-2])
             if node_name in sorted_nodes_names:  # only configurable nodes are in this list
                 node_index_in_graph = sorted_nodes_names.index(node_name)
                 _set_node_final_qc(bit_widths_config,
                                    node,
-                                   node_index_in_graph)
+                                   node_index_in_graph,
+                                   graph.fw_info)
```

Comment:
I thought about it, but I don't want the node to have a specific variable that is dedicated to the kernel, since it is an abstraction of any general op that is not necessarily aware of what kernel is.
We can discuss this offline, but I think that since for now our methodology in the entire code regarding the kernel specifications is done using the fw info, I prefer to keep it that way